handbook.md

```diff
@@ -405,7 +405,7 @@ U malých systémů to znamená přidat k vašemu vývojářskému týmu řekně
 
 Na údržbu softwaru se z hlediska rozpočtu obvykle hledí úplně jinak než na samotný vývoj, což je chyba. Udržovat software znamená jednoduše pokračovat v jeho úpravách podle potřeb uživatelů, které se mění podle zákonů, regulací, politik, doporučených praxí nebo nových technologií. Jsou k tomu potřeba tytéž dovednosti, metody a úkoly, jaké byly potřeba během vývoje. Takže pokud vám někdo navrhuje převést projekt do nějaké speciální fáze provozu a údržby (_operations and maintenance_, O&M), mělo by se vám v hlavě rozsvítit červené světlo.
 
-Obecně řečeno vás agilní tým 5–9 vývojářů bude stát 1–2 miliony dolarů ročně v závislosti na tom, odkud přesně je. Financování se dá postupně navýšit během několika rozpočtových cyklů podle toho, jak se vám bude dařit úspěšně snižovat rizika, kontrolovat výdaje a dodávat iterativně hodnotu koncovým uživatelům.
+Obecně řečeno vás agilní tým 5–9 vývojářů bude stát 12–21 milionů Kč ročně v závislosti na tom, odkud přesně je. Financování se dá postupně navýšit během několika rozpočtových cyklů podle toho, jak se vám bude dařit úspěšně snižovat rizika, kontrolovat výdaje a dodávat iterativně hodnotu koncovým uživatelům.
```

Comment:
Výše máme rozpětí 12–20, navrhuju to sjednotit a dát horní hranici 20 i sem.

Reply:
ok, moje chyba, překlep.